plugins/ragdoll/plugin/commands/sh_fall.lua

```diff
@@ -6,7 +6,12 @@ COMMAND.aliases = { 'fallover', 'charfallover' }
 COMMAND.no_console = true
 
 function COMMAND:on_run(player, delay)
-  delay = math.clamp(tonumber(delay) or 0, 2, 60)
+  delay = math.clamp(tonumber(delay) or 0, 0, 60)
+
+  if delay > 0 and delay < 2 then
+    player:run_command('fall '..tostring(2))
```

Comment:
A better solution would be to just set the delay and let the command run without running it again